### PR TITLE
Update Aspose.PDF

### PR DESCRIPTION
**Pull request by SBalslev**
Review of notice generation revealed that Aspose.Pdf did not resolve to recent changes.
Changed moved the versions in Aspose.Pdf to Apose.PDF.

### CODE REVIEW

File: curations/nuget/nuget/-/Aspose.PDF.yaml [hunk 1]
```diff
@@ -3,9 +3,15 @@ coordinates:
...
+  17.12.0:
+    licensed:
+      declared: OTHER
   21.10.1:
     licensed:
       declared: OTHER
   22.11.0:
     licensed:
       declared: OTHER
+  23.9.0:
```
Comment (SBalslev) [inline curations/nuget/nuget/-/Aspose.PDF.yaml:15]: @capfei the Windows guy didn't notice the two files with same name but different casing 🤣 
I've merged versions from .pdf to .PDF as I realized notice generation is using the uppercase version.

I wanted to clean up so there was only one file but refrained from doing so. There are ~42 such duplicates as I see it, so I assume it is expected.

**Comment by capfei** [comment]
@SBalslev I went to Nuget.org and noticed that the casing changes, at least prior to 22.*.*.
https://www.nuget.org/packages/Aspose.PDF/24.2.0
https://www.nuget.org/packages/Aspose.Pdf/20.12.0
https://www.nuget.org/packages/Aspose.PDF/20.11.0
https://www.nuget.org/packages/Aspose.Pdf/19.12.0
https://www.nuget.org/packages/Aspose.Pdf/17.2.0

**Comment by capfei** [comment]
17.12.0 uses `.Pdf`: https://www.nuget.org/packages/Aspose.Pdf/17.12.0

**Review by capfei — APPROVED** [review]

**Comment by capfei** [comment]
@SBalslev Thank you for the fast update! 😄

**Pull request merged**